Automattic/wp-calypso
# Tests: Move Section-nav tests to single test runner

Here is another one for #3942.  Just needed to delete the make file and register the test in `test.json`
Test works with `.only`
```
$ npm run test-client -- --grep "components section-nav"

> wp-calypso@0.17.0 test-client /Users/jason/Projects/wp-calypso
> NODE_ENV=test NODE_PATH=test:client test/runner.js "--grep" "components section-nav"



  components
    section-nav
      section-nav
        rendering
          ✓ should render a header and a panel
          ✓ should render selectedText within mobile header
          ✓ should render children
        interaction
          ✓ should call onMobileNavPanelOpen function passed as a prop when tapped
          ✓ should call onMobileNavPanelOpen function passed as a prop twice when tapped three times


  5 passing (761ms)
```

**Testing Instructions**

~verify all tests with `npm run test-client`~
verify isolation test by adding `.only` to any of the `describe` blocks and run `npm run test-client`

cc @gziolo
